### PR TITLE
Enumerate exchanges

### PR DESCRIPTION
**Pull request by urdaalex**
Currently returning a list of all exchange names from rabbitmq api.

**Question:** Should just the names of all exchanges be returned? There are other available properties as well.

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -63,6 +63,29 @@ api.declare({
   );
 });
 
+api.declare({
+/*Gets the list of exchanges in the rabbit cluster*/
```
Comment (walac) [inline src/api.js]: nit: use comment format:
```javascript
/*
 * comment
 */
```

Also put it before `api.declare`

File: src/api.js [hunk 2]
```diff
@@ -63,6 +63,29 @@ api.declare({
...
+  method:   'get',
+  route:    '/exchanges',
+  name:     'exchanges',
+  title:    'Rabbit Exchanges',
+  //output:     'rabbit-overview.json',
```
Comment (walac) [inline src/api.js]: nit: removed commented code

File: src/api.js [hunk 3]
```diff
@@ -63,6 +63,29 @@ api.declare({
...
+    '',
+    '**Warning** this api end-point is **not stable**.',
+  ].join('\n'),
+}, async function(req, res) {
+  res.reply(
+    _.map(
+      await this.rabbit.exchanges(),
+      function(elem) {
+        return elem.name;
+      }
```
Comment (walac) [inline src/api.js]: simpler:
```javascript
elem => elem.name
```

File: src/api.js [hunk 4]
```diff
@@ -63,6 +63,29 @@ api.declare({
...
+    )
+  );
+});
+
 /*Gets the namespace, creates one if one doesn't exist*/
```
Comment (walac) [inline src/api.js]: nit: use comment format:
```javascript
/*
 * comment
 */
```

**Comment by Hexcles** [comment]
Hi Alex, #27 was just merged, which introduced JSDoc to `rabbitmanager.js`. Please rebase this PR onto master to solve the conflicts.

As for JSDoc, I see you only added one simple method to `rabbitmanager.js` and it should be rather easy to follow the other examples and add a JSDoc-comment for that method (description is usually copied from the RabbitMQ management API doc directly). But if any question does come up, please let me know and I'm happy to help.

**Pull request merged**